ihe-interop/src/main/java/org/motechproject/ihe/interop/handler/helper/IHEActionHelper.java

```diff
@@ -73,7 +75,12 @@ public void handleAction(Map<String, Object> parameters) throws IOException, Cla
         StreamingTemplateEngine streamingTemplateEngine = new StreamingTemplateEngine();
         Template template = streamingTemplateEngine.createTemplate(new String(ArrayUtils.toPrimitive(templateData)));
         Writable writable = template.make(parameters);
-        LOGGER.info("Template with name {}:\n{}", cdaTemplate.getTemplateName(), writable.toString());
-        iheTemplateService.sendTemplateToRecipientUrl(hl7Recipient.getRecipientUrl(), writable.toString());
+        String result = writable.toString();
+        Pattern regex = Pattern.compile("(?s)<!--.*?-->", Pattern.DOTALL);
```

Comment:
Won't this be a performance suicide given how large our templates are? Wouldn't it make more sense to handle this during save?